package/MDAnalysis/analysis/rms.py

```diff
@@ -437,35 +474,39 @@ def run(self, **kwargs):
             rmsd[k, :2] = ts.frame, trajectory.time
 
             if self.groupselections_atoms:
-                # 1) superposition structures
-                # Need to transpose coordinates such that the coordinate array is
-                # 3xN instead of Nx3. Also qcp requires that the dtype be float64
-                # (I think we swapped the position of ref and traj in CalcRMSDRotationalMatrix
-                # so that R acts **to the left** and can be broadcasted; we're saving
-                # one transpose. [orbeckst])
-                rmsd[k, 2] = qcp.CalcRMSDRotationalMatrix(ref_coordinates_T_64,
-                                                          traj_coordinates.T.astype(np.float64),
-                                                          natoms, rot, weight)
+                # 1) superposition structures Need to transpose coordinates
+                # such that the coordinate array is 3xN instead of Nx3. Also
+                # qcp requires that the dtype be float64 (I think we swapped
+                # the position of ref and traj in CalcRMSDRotationalMatrix so
+                # that R acts **to the left** and can be broadcasted; we're
+                # saving one transpose. [orbeckst])
+                rmsd[k, 2] = qcp.CalcRMSDRotationalMatrix(
+                    ref_coordinates_T_64,
+                    traj_coordinates.T.astype(np.float64), natoms, rot, weight)
                 R[:, :] = rot.reshape(3, 3)
 
-                # Transform each atom in the trajectory (use inplace ops to avoid copying arrays)
-                # (Marginally (~3%) faster than "ts.positions[:] = (ts.positions - x_com) * R + ref_com".)
+                # Transform each atom in the trajectory (use inplace ops to
+                # avoid copying arrays) (Marginally (~3%) faster than
+                # "ts.positions[:] = (ts.positions - x_com) * R + ref_com".)
                 ts.positions -= x_com
-                ts.positions[:] = ts.positions * R  # R acts to the left & is broadcasted N times.
+                # R acts to the left & is broadcasted N times.
+                ts.positions[:] = ts.positions * R
                 ts.positions += ref_com
 
                 # 2) calculate secondary RMSDs
                 for igroup, (refpos, atoms) in enumerate(
-                        zip(groupselections_ref_coords_T_64, self.groupselections_atoms), 3):
-                    rmsd[k, igroup] = qcp.CalcRMSDRotationalMatrix(refpos,
-                                                                   atoms['mobile'].positions.T.astype(np.float64),
-                                                                   atoms['mobile'].n_atoms, None, weight)
+                        zip(groupselections_ref_coords_T_64,
+                            self.groupselections_atoms), 3):
+                    rmsd[k, igroup] = qcp.CalcRMSDRotationalMatrix(
```

Comment:
I think we need to check what we're doing here (sorry for commenting here, it's not really part of this issue): The subselection RMSDs *also do an implicit rotation* and from the docs this is not clear. In fact, I think when we (I, probably...) implemented this experimental feature, the idea was to calculate RMSDs of sub-groups in the context of the overall superposition.

Probably requires raising a new issue.